src/include/ipc/info.h

```diff
@@ -139,6 +140,15 @@ struct sof_ipc_cc_version {
 	char desc[]; /* null terminated compiler description */
 } __attribute__((packed));
 
+/* extended data: Probe setup */
+struct sof_ipc_probe_support {
+	struct sof_ipc_ext_data_hdr ext_hdr;
+
+	uint32_t probe_points_max;
+	uint32_t injection_dmas_max;
+} __attribute__((packed));
+
 extern const struct sof_ipc_cc_version cc_version;
+extern const struct sof_ipc_probe_support probe_support;
```

Comment:
Hmm, these two externs should not be in IPC UAPI headers but better in local ipc header for correct scope.

Reply:
it also seems that they should be in ```__attribute__((section(".fw_ready")))``` alongside the SRAM windows too.

Reply:
@lgirdwood `ipc/info.h` header is outside `uapi` tree, which it should probably be part of. Should I change this in the meantime as well?

Reply:
@akloniex sorry not following, why do we need to move info.h ? It;s kernel only data and part of IPC.

Reply:
@lgirdwood I initially misinterpreted your intent. I believe that current change handles that just right.